mobile/src/main/java/org/openhab/habdroid/ui/widget/ExtendedSpinner.java

```diff
@@ -0,0 +1,42 @@
+package org.openhab.habdroid.ui.widget;
+
+import android.content.Context;
+import android.support.v7.widget.AppCompatSpinner;
+import android.util.AttributeSet;
+
+/*
+ * An extended version of the Spinner class, which allows getting
+ * callbacks for selection updates no matter whether the selection
+ * has actually changed or not.
+ */
+public class ExtendedSpinner extends AppCompatSpinner {
+    public interface OnSelectionUpdatedListener {
+        void onSelectionUpdated(int position);
+    }
+
+    private OnSelectionUpdatedListener mOnSelectionUpdatedListener;
+
+    public ExtendedSpinner(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public ExtendedSpinner(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    public void setOnSelectionUpdatedListener(OnSelectionUpdatedListener l) {
+        mOnSelectionUpdatedListener = l;
```

Comment:
This really looks like `mOnSelectionUpdatedListener` is an int and set to `one`.

Reply:
I can rename it to `listener`.

Reply:
Go for it :+1: